src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -25,7 +25,7 @@ function getAppointmentType(appt) {
   return APPOINTMENT_TYPES.vaAppointment;
 }
 
-function getTypeOfCareById(id) {
+export function getTypeOfCareById(inputId) {
```

Comment:
Is this function the same as 'getTypeOfCare(serviceType)' in ../vaos/utils/appointment.js?

```
export function getTypeOfCare(serviceType) {
  const allTypesOfCare = [
    ...TYPES_OF_EYE_CARE,
    ...TYPES_OF_SLEEP_CARE,
    ...AUDIOLOGY_TYPES_OF_CARE,
    ...TYPES_OF_CARE,
  ];

  return allTypesOfCare.find(
    care =>
      care.idV2 === serviceType ||
      care.ccId === serviceType ||
      care.id === serviceType,
  );
}
```

Reply:
Yeah that is why Jennie recommended I delete it. I had created it because the ones in the transformers were not exported. Her suggestion was to go ahead and export them so, we could have one instance of the logic

Reply:
I'm not sure I understand. Why not use the one in ../vaos/utils/appointment.js? So, delete the two exported functions and import the original.

Reply:
the one in /vaos/utils/appointment.js was the function I wrote. The two in the transformer and transformer.v2 were the originals. Jennie suggested I delete the one I wrote and use the existing ones. But, I can put the function back into utils/appointment.js and change the existing transformer and transforner.v2 to use that.

Reply:
Ah! I wont keep holding this up. It makes more since to me to change it like you originally had it since the function is an utility function. It's up to you though.

Reply:
yeah no worries. I can make that change